src/postgraphql/graphiql/graphiql.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react'
+import { render } from 'react-dom'
+import GraphiQL from 'graphiql'
+import { buildClientSchema, introspectionQuery } from 'graphql'
+import 'graphiql/graphiql.css'
```

Comment:
I am using css (modules) for the components. Are you still opposed? I might have made things a bit too complicated. I will push the new changes and you can evaluate if this is ok.